automated install/basic-install.sh

```diff
@@ -685,8 +696,15 @@ find_IPv4_information() {
     local route
     local IPv4bare
 
-    # Find IP used to route to outside world by checking the route to Google's public DNS server
-    route=$(ip route get 8.8.8.8)
+    # Test if IPv4 is routable
+    if ip route get 8.8.8.8 &>/dev/null; then
```

Comment:
Here you run the same code twice (`ip route get 8.8.8.8`).
Would it make sense to use `route$(ip route get 8.8.8.8 &>/dev/null)` and then check it `route` is empty?